frontend/src/pages/about/About.tsx

```diff
@@ -0,0 +1,310 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Typography, Box, Card } from '@material-ui/core';
+
+import { ContentHeader } from 'src/components';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '100%',
+    padding: '32px 0px 32px 0px',
+    justifyContent: 'center',
+  },
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    padding: 8,
+  },
+  card: {
+    border: '1px solid #DCD8CB',
+    boxShadow:
+      '0px 0px 8px rgba(0, 0, 0, 0.02), 0px 2px 4px rgba(0, 0, 0, 0.05)',
+    borderRadius: 4,
+    padding: 8,
+    width: '100%',
+    background: '#FFFFFF',
+  },
+}));
+
+const PeopleCard = ({
+  name,
+  image,
+  institution,
+  role,
+  description,
+}: {
+  name: string;
+  image: string;
+  institution: string;
+  role: string;
+  description: string;
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.card} style={{ maxWidth: 320 }}>
```

Comment:
`classes.card` is mostly redundant with the built-in style of `<Card>`.
```suggestion
    <Card style={{ maxWidth: 320, padding: '16px' }}>
```